modules/serverless-quarkus-functions-overriding-liveness-readiness.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies
+//
+// * functions/reference/serverless-developing-quarkus-functions.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-quarkus-functions-overriding-liveness-readiness_{context}"]
+= Overriding liveness and readiness probe values
+
+You can override `liveness` and `readiness` probe values for your Quarkus functions. This allows you to modify the timing of health checks performed on the function.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving are installed on the cluster.
+* You have installed the Knative (`kn`) CLI.
+* You have created a function by using `kn func create`.
+
+.Procedure
+
+. Override the `/health/liveness` and `/health/readiness` paths with your own values. You can do this either by changing properties in the function source or by setting the `QUARKUS_SMALLRYE_HEALTH_LIVENESS_PATH` and `QUARKUS_SMALLRYE_HEALTH_READINESS_PATH` environment variables on `func.yaml` file.
++
+To override the paths using the function source, update the path properties in the `src/main/resources/application.properties` file:
++
+[source]
+----
+quarkus.smallrye-health.root-path=/health <1>
+quarkus.smallrye-health.liveness-path=alive <2>
+quarkus.smallrye-health.readiness-path=ready <3>
+----
+<1> The root path, which is automatically prepended to the `liveness` and `readiness` paths.
+<2> The liveness path, set to `/health/alive` here.
+<3> The readiness path, set to `/health/ready` here.
++
+To override the paths using environment variables, define the path variables in the `build` block of the `func.yaml` file:
++
+[source,yaml]
+----
+build:
+  builder: s2i
+  buildEnvs:
+  - name: QUARKUS_SMALLRYE_HEALTH_LIVENESS_PATH
+    value: alive <1>
+  - name: QUARKUS_SMALLRYE_HEALTH_READINESS_PATH
+    value: ready <2>
+----
+<1> The liveness path, set to `/health/alive` here.
+<2> The readiness path, set to `/health/ready` here.
```

Comment:
Could be better emphasis these as two disjoint option? For instance make them to be two list items (a and b), or maybe tabs?

Reply:
```
Override the /health/liveness and /health/readiness paths with your own values. You can do this either by changing properties in the function source (a) or by setting the QUARKUS_SMALLRYE_HEALTH_LIVENESS_PATH and QUARKUS_SMALLRYE_HEALTH_READINESS_PATH environment variables on func.yaml file (b).

a) To override the paths using the function source, update the path properties in the src/main/resources/application.properties file:

b)To override the paths using environment variables, define the path variables in the build block of the func.yaml file:
```

Reply:
Thanks for the review @matejvasek !
Made the necessary modifications.